textdb/textdb-dataflow/pom.xml

```diff
@@ -41,6 +41,21 @@
             <artifactId>textdb-storage</artifactId>
             <version>${textdb.version}</version>
         </dependency>
+
+
+        <dependency>
+            <groupId>edu.stanford.nlp</groupId>
+            <artifactId>stanford-corenlp</artifactId>
+            <version>3.6.0</version>
+        </dependency>
+        <dependency>
+            <groupId>edu.stanford.nlp</groupId>
+            <artifactId>stanford-corenlp</artifactId>
+            <version>3.6.0</version>
+            <classifier>models</classifier>
+        </dependency>
+
+
         <dependency>
             <groupId>org.mockito</groupId>
```

Comment:
@sam0227 @chenlica .. In `NamedEntityExtractorTest` we can use mockito since `NamedEntityExtractor` has dependency on `IOperator`. So, we can mock `IOperator` instead of creating `ScanBasedSourceOperator`.